blocks/bricks/recurrent.py

```diff
@@ -300,6 +301,109 @@ def apply(self, input_=None, state=None, mask=None):
         return next_state
 
 
+class LSTM(BaseRecurrent, Initializable):
+    u"""Long Short Term Memory.
+
+    Every unit of an LSTM is equipped with input, forget and output gates.
+    This implementation aims to do as many computations in parallel as
+    possible and expects the last dimension of the input to be four times
+    the output dimension.
+
+    Parameters
+    ----------
+    dim : int
+        The dimension of the hidden state.
+
+    Notes
+    -----
+    See :class:`.Initializable` for initialization parameters.
```

Comment:
There's no easy way that I know of. You can build the documentation
yourself if you have sphinx and the RTD theme installed. It's quite
straightforward but I haven't added the instructions to the docs yet. For
now just assume it's okay, we'll fix it if they don't turn out right.
On Feb 10, 2015 12:13 PM, "Philemon Brakel" <notifications@github.com>
wrote:

> In blocks/bricks/recurrent.py
> <https://github.com/bartvm/blocks/pull/234#discussion_r24430002>:
>
> > +class LSTM(BaseRecurrent, Initializable):
> > +    u"""Long Short Term Memory.
> > +
> > +    Every unit of an LSTM is equipped with input, forget and output gates.
> > +    This implementation aims to do as many computations in parallel as
> > +    possible and expects the last dimension of the input to be four times
> > +    the output dimension.
> > +
> > +    Parameters
> > +    ----------
> > +    dim : int
> > +        The dimension of the hidden state.
> > +
> > +    Notes
> > +    -----
> > +    See :class:`.Initializable` for initialization parameters.
>
> I changed it but I'm not sure how the references will appear in the docs.
> What is the easiest way to check that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bartvm/blocks/pull/234/files#r24430002>.
>